srcpkgs/fatsort/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'fatsort'
+pkgname=fatsort
+version=1.6.2.605
+revision=1
+build_style=gnu-makefile
+hostmakedepends="help2man"
+short_desc="Utility that sorts FAT12, FAT16, FAT32 and exFAT partitions"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
+license="GPL-2.0-or-later"
+homepage="https://fatsort.sourceforge.io/"
+distfiles="https://iweb.dl.sourceforge.net/project/fatsort/fatsort-${version}.tar.xz"
```

Comment:
`distfiles="${SOURCEFORGE_SITE}/project/${pkgname}/${pkgname}-${version}.tar.xz"`

You can leave `fatsort` instead of `${pkgname}` if you want, but this is the most general approach. Definitely prefer `$SOURCEFORGE_SITE` regardless of the `$pkgname` decision.

Reply:
done, decided to keep `fatsort` though